developer/src/kmcmpdll/Compiler.cpp

```diff
@@ -3327,7 +3327,9 @@ DWORD WriteCompiledKeyboard(PFILE_KEYBOARD fk, HANDLE hOutfile)
     return CERR_SomewhereIGotItWrong;
   }
 
-  SetChecksum(buf, &ck->dwCheckSum, (DWORD)size);
+  if (ck->dwFileVersion < VERSION_160) {
+    SetChecksum(buf, &ck->dwCheckSum, (DWORD)size);
+  }
```

Comment:
```suggestion
  if (ck->dwFileVersion < VERSION_160) {
    SetChecksum(buf, &ck->dwCheckSum, (DWORD)size);
  }
  else {
    ck->dwCheckSum = 0; // checksum is deprecated for 16.0+
  }
```

This may be done earlier by zeroed structures but I like the clarity!